notebooks/least_core_basic.ipynb

```diff
@@ -0,0 +1,811 @@
ixhyichdlcxWcBLhJMS6EMfOujknvYRa4+EeQ8IRCkxKqZ8G/hvAAP+DiPzNDfdfAf4+MF7O838XkX9e3vf/AP4axWnn3xSRPzqq9fZ4ANI848vvfJvUZkzUTj3o1TlwMsm4Zq/ydv4WV/M3uWbfxZKj0Vwyj/GF+Cd5yjzLE8FTxGqkxMxB0H2QwpJsKINTmGqFoLrXswODfKVZItPZUb4SQHemzsxLl1l89QxKCZMfmuXcZ25SPdvd9jFROs+l+f+d8c5r9MMzvH3h11mtP7f1zEohKsISYQ/jxIe4EYEqXX/pUszGaSP3jU4zLiXMV9AuGxG1UrTsrjRG0CMiVLSt62qdu+qeYtb6eVDbtL9WgDEYs3a/CGAt2cpKeQOU0uhKTFCpoqIIFYZoczhf/KJ8UIpSjobFpgnZygpa94hPGVSgie3Ktl3Ywnxl3Zk2QZMG42XJ2iQ9c5o+Z+lxhr5MkbhxXBZsm39VTF+ffzUIcs+74frug5neNrdk+9e7cSCzRWfBjQObdYOde4hboX2ozFUuV0XJ2qAD23I8LGFLlivknQ0CknFEpwq3Ue3cwrCcbSAkBfWTKCBtjS+BOzgKsWmcxI6jKRyP1XCJqBSbchfj5JjEBeRuszDU30IoGkzbTjAyuhSGAqiGMF4rrg+mxWF5Wf4d0PZfKVCBoIMcDmEMuq7jbLp5Gz6ctp2QNbJfWNdxdrCv2FfH2Q0nKvbacXZ0e2/kodmmeQ4OhSucf8qilEMhpYwkODFYF5O4OpmtFs5NCofhTkUkzxEJTEopA/x3wE8CN4FvKKV+X0ReH5nt/wn8roj8LaXUB4B/DjxeXv8V4HngAvCvlVLvE9nliMXj2SO5s3zt+qus9Fucbkw86NU5EHLJuGbfLUK58ze5Zt8hJ0ehuGSu8CPRj/N08CxPBE9TUVsc5UjhWIpaAdqCDY9WWBJnyVotkvl58m4PHRiCWn3PZXCanDhYpR7dxahsR/lKItB67xQzL15m9eokOsqZ/tQtpj99i+jU9l1vtEuYXvw3TC//OwTDzdM/x9z45xH1AA2lSpflcYeUtSF2KDqZoesqGxGqknUClymdWOvnKe4LsvZw2kD4UruMTHTKbO+u2ui+2pBt5XSMVSE2DbDLZujkyoMKqjqGqjTRcQxhiN5jd8L1750jSJYwrZvopRtE/Vliu0zslstMpOV1r19QZMEpkmCSVvXpdeVrRaD2qW0FNgPU6G15315xVo0ITqOB7CMDl7Qc3NxjMGP7AVlrQ8h7uvuDPWWKgYvZ6JwanRaMDFIG00cHQdEWg57IHvjZeGcV6Uq8oYxtzYGUtdb/XpV2RKcKB9L4+xZK4WithC1spA/1YGu0BC7xJXCHgiOkbyfo24lCbApa1IJFItNGUFgXHazYZDc4jDaUog3FpMF0u82+QCuojIhCY5VSMNogFA3Eo+DhFCLXdZytHWLH2RGhym4QsradtsF9m61Gm6btu+PsiGt2uF3fWDq+VTn5ltMerhMWDxvrRaQ1J9L9RSRf3HUQHNW7+EngbRF5B0Ap9TvALwCjApMAY+X1U8Dt8vovAL8jIgnwrlLq7XJ5XzmKFfc82jhxvHLj+9xemWN6bOr+Dzim5JJzw77H22XJ23v2KjkZCsVFfZnPRj82FJSqqrb9ggRMXxOvGpRTuECwR7gtdllKurxMOjePtRYTR4T7KINby1eaQ+GKfCV3b5FFHCy9cZqZFy/RvT1GUE+5+OPvcuaFOwTVexywiTDR/jYX5/8ZUb7MQvPj3D79c2TBw+eI24QyOGNwVDj4Q1pQkm/pqNrSfbWVE6sUt4zrEbqVDSWH6Z7yr6wq86pMjAsqiKkUlzrGmULMcyYubxcOLdERQbpEmMwT9ueIkjnC/gKK9edTUjNGGk7RqT7O4oh4lASTZOH4gxUrN6CNoI2FQwh4HpaTbDrTPipkbXOmfsPAJe8GpFm8bprbQ07C7vKvLJpCYFU2J+3WSDs1knadpFUla1dYJ6ApITpVdFwbe2pppHytEJLCZvLIDXgUDqOTogTORbST8yR27OSXb50AHCH9fJJ+PolRaelsWiidTYp8K7HJuQ1Ooi1K0foj4lF+D8FoVBBqVEbcRSMOo8qIw+hhVlePCes6znI4HWftBpfVelftVuLWhmnlviDvhJu60+62YYcy25yk2Dhti3JyH85+EEjhQFKuPNE2KGcrnfAuwEqElRBxBodBxDzQkw5ZGmIee/i3RUd1FHoRuDFy+ybwqQ3z/JfAv1RK/adAHfiJkcd+dcNjL258AqXUXwf+OsCVK1cOZKU9jzYiwmt3rvLO4k2mm6cf9OrsCis5N+y1ssvbG7yXXyUlBeCCvsRnos/zVPAsTwbPUFM76GYwEJZaBpUrJBBcdFQ7x7IMbn6BZGUZhcJUq4R7tqRLma80X+YraXJbuW++kssVC98uOsIli1XiyR6P/ewPmProzJYd4UapJLe5PPdPaPau0o0v8u65v0Kn+sTmNRNIxOLEYZQmUBrjD4rvi6gAawKsuYc4uueFD/KvsvWC1QbX1UZRSw3msUlxn00x/TaaHMN6Z9ZGsnCcNJqkbS7Qbz5HFp0mjU+XDqQJRB8fAelBMlpOEhyCdDk8G3+f7JP15SUbBjN9hesrslWNy4oBjc0NzgaIbPwcHVHYJgpXGQtniE6vEkerROEqcbhCFLVRgSrcFcZAomDewLIup224HF43W0zbeGn27AB9EPgSuGOCc6g0RdKELMnI0wpBagmyFkE2h0oSSCzSd5BYyLYRmhXry84atc2laJUNDiO/b3zkUJqiG+VhnbBwat12fP1Jintv70dPXBSO2xHha4/5V56HE/Oj2+eyPiwcp6PUXwX+noj8f5RSPwz8llLqgzt9sIj8HeDvALzwwgteFvbsm7fnb/DqzFWmx6ZQx/xAxorllr3O27ZwKL2bv01KUaZ1Xl/kk9HneDp4lifNM9T1Lhw/AiZRxK0AlZXCUnw0Py9xlmy1
```

Comment:
In the current plot it looks like the highest budget actually leads to lower accuracy, right? I wonder if this is related to the fact that the dataset is quite simple. It feels like more complex data would have scores more dependent on budget.

Reply:
Yeah, you're right. I changed the description after that plot to describe two reasons why this could be the case.